public/massmint/template.csv

```diff
@@ -1,5 +1,5 @@
-file,name,description,price
-art1.png,Art #1,Description for the Art #1,1
-art2.png,Art #2,Description for the Art #2,1
-art3.png,Art #3,Description for the Art #3,1
-art4.png,Art #4,Description for the Art #4,1
+file,name,description,attributes,price
+art1.png,Art #1,Description for the Art #1,"color:white;expression:happy",1
+art2.png,Art #2,Description for the Art #2,"color:blue;expression:shy",1
```

Comment:
why does it have quotes? (i mean `"`)
Is there any special reason for that ?

Reply:
No special reason and I removed the quotes. It works the same